canopeum_frontend/src/pages/Login.tsx

```diff
@@ -1,150 +1,116 @@
       </div>
 
-      <div className='col-10 col-sm-6 col-md-8 col-xl-6 d-flex flex-column gap-4'>
+      <form
+        className='col-10 col-sm-8 col-xl-6 d-flex flex-column gap-4'
+        onSubmit={handleSubmit(onSubmit)}
+      >
         <div className='w-100'>
-          <label htmlFor='email-input'>{translate('auth.email-label')}</label>
+          <label htmlFor='email'>{translate('auth.email-label')}</label>
           <input
-            aria-describedby='email'
+            aria-describedby='emailHelp'
             className={`form-control ${
-              emailError
-                ? 'is-invalid'
+              touchedFields.email && errors.email
+                ? formClasses.invalidFieldClass
                 : ''
             }`}
-            id='email-input'
-            onBlur={() => validateEmail()}
-            onChange={event => setEmail(event.target.value)}
-            type='email'
+            {...register('email', {
+              required: { value: true, message: translate('auth.email-error-required') },
+            })}
           />
-          {emailError === 'required' && (
-            <span className='help-block text-danger'>
-              {translate('auth.email-error-required')}
-            </span>
-          )}
+          {errors.email && <span className='help-block text-danger'>{errors.email.message}</span>}
         </div>
 
         <div className='w-100'>
           <label htmlFor='password-input'>{translate('auth.password-label')}</label>
           <input
             className={`form-control ${
-              passwordError
-                ? 'is-invalid'
+              touchedFields.password && errors.password
+                ? formClasses.invalidFieldClass
                 : ''
             }`}
-            id='password-input'
-            onBlur={() => validatePassword()}
-            onChange={event => setPassword(event.target.value)}
             type='password'
+            {...register('password', {
+              required: { value: true, message: translate('auth.password-error-required') },
+            })}
           />
-          {passwordError === 'required' && (
-            <span className='help-block text-danger'>
-              {translate('auth.password-error-required')}
-            </span>
+          {errors.password && (
+            <span className='help-block text-danger'>{errors.password.message}</span>
           )}
         </div>
 
         <div>
-          <Checkbox
-            checked={rememberMe}
-            id='remember-me'
-            onChange={(_value, isChecked) => setRememberMe(isChecked)}
-            value='remember-me'
-          >
-            Remember Me
-          </Checkbox>
+          {/* TODO: Make a component, this is extracted from the Checkbox  */}
+          {/* component until everything uses React Hook Forms */}
+          <div className='form-check'>
+            <input
+              className='form-check-input'
+              id='checkbox-input'
```

Comment:
This is too vague of an ID, please keep the same id as before (and update the linked id on the label as mentioned in another comment)